include/slang-rhi.h

```diff
@@ -2942,6 +2942,8 @@ class IDevice : public ISlangUnknown
     virtual SLANG_NO_THROW bool SLANG_MCALL hasCapability(Capability capability) = 0;
     virtual SLANG_NO_THROW bool SLANG_MCALL hasCapability(const char* capability) = 0;
 
+    virtual SLANG_NO_THROW uint32_t SLANG_MCALL getOptixVersion() = 0;
```

Comment:
The optix version is already available in `DeviceInfo::optixVersion` so no need for this.